x/tally/keeper/endblock.go

```diff
@@ -72,6 +73,7 @@ func (k Keeper) ProcessTallies(ctx sdk.Context, coreContract sdk.AccAddress) err
 		k.Logger(ctx).Error("[HALTS_DR_FLOW] failed to unmarshal tally-ready data requests", "err", err)
 		return nil
 	}
+	telemetry.SetGauge(float32(len(tallyList)), "seda_tally_end_block_number_of_data_requests_to_tally")
```

Comment:
I think a counter would be more appropriate for this? Like: https://github.com/cosmos/cosmos-sdk/blob/8bfcf554275c1efbb42666cc8510d2da139b67fa/x/staking/keeper/msg_server.go#L366

And I think we should increment one by one every time a tally is completed?

Reply:
We don't want a counter here. This one is to emit the number of data request tallies processed in the last block.

You could then use Prometheus to get an avg of dr tallies processed over a period of time. Or just have a live view/graph etc.

Reply:
Ah understood. thanks